meta.js

```diff
@@ -27,12 +27,12 @@
 // @description:it	Blocca gli annunci di popup nelle pagine internet
 // @description:vi	Chặn quảng cáo popup trên các trang web
 // @description:es	Bloquea popups de anuncios en sitios web
-// @version 2.0
-// @adg:permanent	true
+// @version 2.1
+// @adg:permanent	[ADG_PERMANENT]
```

Comment:
It'd be better to simply remove this attribute (deprecated in newer AG versions)